items.lua

```diff
@@ -0,0 +1,620 @@
+return {
+    ['testburger'] = {
```

Comment:
Do we need to keep the test items?

Reply:
I pretty much kept all the standard ox items including the test burger as it showcases pretty much everything that can be done to an item. If we want we can remove it but that's why I kept it.